test/test_wiki_trac_rst_convert.py

```diff
@@ -0,0 +1,105 @@
+import unittest
+
+from wiki_trac_rst_convert import convert_content
+
+
+class TracRstToVanillaRst(unittest.TestCase):
+    """
+    Test conversion of content from Trac-flavored reStructuredText to
+    vanilla reStructuredText, that is supported by GitHub
+    """
+
+    def assertConvertedContent(self, expected: str, source: str):
+        """
+        Run the Trac RST `source` through the converter,
+        and assert that the output equals `expected`.
+        """
+        self.assertEqual(expected, convert_content(source))
+
+    def test_empty(self):
+        """
+        An empty string is appended a blank line.
+
+        A file not ending in a newline character is not POSIX compliant,
+        and may result in complaints from programs, like `git diff`
+        saying "\ No newline at end of file".
+        https://stackoverflow.com/a/729795/235463
+        """
+        self.assertConvertedContent('\n', '')
+
+    def test_newline(self):
+        """
+        A newline should not be appended another newline.
```

Comment:
No need for `should` ... as this is a test docstring :)

```suggestion
        A newline will not get appended another newline.
```

Reply:
D'oh. That one got through.

(Should I avoid comments like this in order not to spam you?)

Reply:
It's ok... there are other comments and I will be notified anyway. Don't worry. Thanks